chcemvediet/apps/accounts/tests/test_backend.py

```diff
@@ -0,0 +1,82 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from poleno.utils.urls import reverse
+
+
+class AdminLoginAsBackend(TestCase):
```

Comment:
@martinmacko47 Zatial som to spravil v `chcemvediet.accounts` appke, len ako hruby navrh riesenia pre diskusiu riesenia a opravu pripadnych chyb. Este to mam v plane presunut do vseobecnej libky. Taktiez testy este podla potreby prerobim aby fungovali vo vseobecnej appke `poleno`.

Reply:
@viliambalaz Mas teraz rozrobene dva backendy. Jeden v `poleno/utils/backends.py` a druhy v `chcemvediet/apps/accounts/backends.py`. Aby v tom nebol chaos, nechaj z nich iba jeden a ten doladime. Potom ked bude fungovat, mozeme ho vysunut do separatnej libky.